permissions/component-remoting-opentelemetry-parent.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "remoting-opentelemetry-parent"
+github: "jenkinsci/remoting-opentelemetry-plugin"
+paths:
+- "io/jenkins/plugins/remoting-opentelemetry-parent"
```

Comment:
if you're using JEP-229, you should just change your path in the plugin file to use a * after opentelemetry

otherwise it won't work

https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/plugin-remoting-opentelemetry.yml#L13-L14

see https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/pom-bom.yml#L7